config/samples/dbaas.redhat.com_v1alpha1_crunchybridgeconnection.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: dbaas.redhat.com/v1alpha1
+kind: CrunchyBridgeConnection
+metadata:
+  name: crunchybridgeconnection-sample
+spec:
+  # Add fields here
+  foo: bar
```

Comment:
Might be worth updating with a correct example

Reply:
updated example as seen. 
``` apiVersion: dbaas.redhat.com/v1alpha1
kind: CrunchyBridgeConnection
metadata:
  name: crunchybridgeconnection-sample
spec:
  instanceID: hw4j2fw33zgb3mx5lpoiosj3va
  inventory:
    name: crunchybridgeinventory-sample
```